beaker-vue/src/components/notebook/BeakerNotebookToolbar.vue

```diff
@@ -243,6 +243,14 @@ const handleExport = (format: string, mimetype: string) => {
         });
     }
 
+    if (format === "full_context_notebook") {
+        const history = await session.executeAction("get_agent_history", {}).done;
+        console.log(history);
+        tempNotebook.content.metadata.message_history = {
+            full_history: history.content.return
+        }
```

Comment:
This introduces two brand new terms, "message_history" and "full_history", which I believe to be essentially synonyms with the existing term "chat_history".

Is there any reason to not just use `chat_history` here and bypass the nesting. Something like `tempNotebook.content.metadata.chat_history = history.content.return`?